packages/twenty-front/src/modules/object-record/record-field/meta-types/input/components/MultiItemFieldInput.tsx

```diff
@@ -46,6 +47,7 @@ export const MultiItemFieldInput = <T,>({
   formatInput,
   renderItem,
   hotkeyScope,
+  isPhoneInput = false,
```

Comment:
This breaks the generic nature of this component, we should find a way to have the parent component, renderItem or another specific part handle the change.

It would probably require to apply this refactor to other multi item input components (links and emails) as well.